src/connectRoutes.js

```diff
@@ -317,6 +329,9 @@ export default (
     }
 
     if (typeof window !== 'undefined') {
+      // `kind` is typed as ?string which will not throw if passed to .test.
+      // Typing kind as string threw other flow errrs in the codebase
+      // $FlowIssue
```

Comment:
Yeah, I was thinking it should probably always be defined but didn't want to change something like that since it seems unrelated to this PR